dev/tests/behavioral/behat.yml

```diff
@@ -15,3 +15,4 @@ default:
                      "version": ""
```

Comment:
Sorry, but unfortunately this commit 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to pass all tests. 1 features failed.
Failed suites:
- Behat tests [1]
http://localhost/test-run/317/